cypress/integration/advancedOption.test.ts

```diff
@@ -0,0 +1,71 @@
+import { PoolName } from "../../src/constants"
+
+context("Advanced option test", () => {
+  // have two seperate maps here since the naming convention is different throughout the page
+  const pools = ["BTC Pool V2", "Stablecoin Pool V2"]
+  const advancedOptionValues = ["0.1", "20", "20"]
+
+  beforeEach(() => {
+    const host = Cypress.env("DAPP_HOST") as string
+    cy.visit(`${host}#/pools`)
+    cy.waitForReact()
+    cy.wait(3000)
+  })
+
+  function advancedOptionTest(poolName: PoolName) {
+    const host = Cypress.env("DAPP_HOST") as string
+    cy.visit(`${host}#/pools`)
+    cy.wait(3000)
+    cy.contains(poolName)
+      .parents(".poolOverview")
+      .within(() => {
+        cy.get("button").contains("Withdraw").click()
+      })
+    cy.wait(10000)
+
+    cy.get("[data-testid=optionContainer]").should("exist")
+    cy.get("[data-testid=optionContainer]").then((advancedOptions) => {
+      if (advancedOptions.find("[dat-testid=tableContainer]").length > 0) {
+        cy.get("[dat-testid=tableContainer]").then(($table) => {
+          const tableDisplay = $table.css("display")
+          const isTableVisible = tableDisplay !== "none"
+
+          if (!isTableVisible) {
+            cy.get("[data-testid=title]").click()
+          }
+        })
+        cy.get("[data-testid=infiniteApproval]")
+          .find('[type="checkbox"]')
+          .check({ force: true })
+
+        cy.get("[data-testid=inputGroup]").each(($inputGrop, index) => {
+          const display = $inputGrop.parent().css("display")
+
+          if (display !== "none") {
```

Comment:
We should be caution of using conditional statements in Cypress tests.

https://docs.cypress.io/guides/core-concepts/conditional-testing#The-DOM-is-unstable

Maybe adding data tag to "inputGroup" parent (if possible) then checking like

`cy.get(['data-testid=inputGroupContainer']).should('not.exist')` Something like this could work